src/js/popup.js

```diff
@@ -333,6 +340,24 @@ $(document).ready(function () {
         });
     });
 
+    $('#rate').click(function () {
+        dmUIClick('rate');
+        chrome.runtime.sendMessage({
+            JDTIfunction: 'setRateStatus'
+        }, function (response) {
+            window.open('https://chrome.google.com/webstore/detail/jira-template-injector/hmhpegjieopgbdmpocdmfkafjgcdmhha/reviews?hl=en', '_blank'); // Open extension user reviews page
```

Comment:
Using chrome.runtime.id function to get extension ID. 
However, the extension ID assigned for JTI in the official extension store is different from the one for unpacked extension package.